### PR TITLE
Update Jakarta Persistence API 3.2.0-M2

### PR DESCRIPTION
**Pull request by Riva-Tholoor-Philip**
Update Jakarta Persistence API 3.2.0-M2
for #27846

### CODE REVIEW

**Comment by Riva-Tholoor-Philip** [comment]
#11428

**Comment by Riva-Tholoor-Philip** [comment]
#build

File: dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml [hunk 1]
```diff
@@ -1316,6 +1316,11 @@
       <artifactId>jakarta.persistence-api</artifactId>
       <version>3.2.0-M1</version>
     </dependency>
+    <dependency>
```
Comment (tkburroughs) [inline dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml]: This should replace the above entry, not result in a new entry... please remove the one for M1 above.

**Review by tkburroughs — CHANGES_REQUESTED** [review]
`check_this_in_if_it_changes/pom.xml` should be an update from M1 to M2, not the addition of a whole new M2 section.  Please update.

**Comment by Riva-Tholoor-Philip** [comment]
#build

**Comment by Riva-Tholoor-Philip** [comment]
#libby

File: dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml (hunk 1 above)
Comment (Riva-Tholoor-Philip) [reply dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml]: made requested change

**Review by tkburroughs — APPROVED** [review]
Changes look good; approving.  Now we just need to wait for the build to complete: https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_Gbvz4OFSEe62Qc2Tit17uw

**Pull request merged**